src/progress_indicators-bars.adb

```diff
@@ -0,0 +1,37 @@
+--  SPDX-License-Identifier: Apache-2.0
+--
+--  Copyright (c) 2021 onox <denkpadje@gmail.com>
```

Comment:
The standard here should be:

```
Copyright (c) 2021 The progress_indicators authors
```

and then do an CONTRIBUTORS.TXT file at the repo root.

Reply:
I've added a CONTRIBUTORS.md file so that it gets rendered by GitHub.